libraries/Microsoft.Bot.Builder/StateTurnContextExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder
```

Comment:
>Microsoft.Bot.Builder [](start = 10, length = 21)

hmmm..I think we had a convention to put this into a unique namespace, so that you can opt into these extensions or not.  We should talk about that

Reply:
Hmm, ok

Reply:
Removing this - pretty marginal value.